content/docs/reference/addon-api/addon.tab/_index.md

```diff
@@ -949,7 +939,10 @@ Adds a new block to the Debugger category in the block palette.
 Removes a block that was previously added to the Debugger category in the block palette.
 
 ### `addon.tab.loadScript`
-**In most cases, you should use the [`userscripts` property of the addon manifest](/docs/reference/addon-manifest/#userscripts-and-userstyles) instead.**
+
+{{< admonition warning >}}
+In most cases, you should use the [`userscripts` property of the addon manifest](/docs/reference/addon-manifest/#userscripts-and-userstyles) instead.

```

Comment:
Should we officially deprecate this and also mention not importing 3rd-party libraries with it?

Reply:
I'm not sure, but we could write down that usually we prefer addons that don't import any libraries.

Reply:
```suggestion
Use the standard browser import instead.
```

Reply:
> I'm not sure, but we could write down that usually we prefer addons that don't import any libraries.

Sure but I think that fits better in the best practices page and I want to try to keep this PR to 1 file.